documentation/asciidoc/accessories/display/display_intro.adoc

```diff
@@ -1,45 +1,86 @@
 == Raspberry Pi Touch Display
 
-The https://www.raspberrypi.com/products/raspberry-pi-touch-display/[Raspberry Pi Touch Display] is an LCD display which connects to the Raspberry Pi through the DSI connector. In recent versions of Raspberry Pi OS, both the HDMI and LCD display outputs can be used at the same time.
+The https://www.raspberrypi.com/products/raspberry-pi-touch-display/[Raspberry Pi Touch Display] is an LCD display which connects to the Raspberry Pi through the DSI connector. If your Raspberry Pi has both connectors, you can use the micro HDMI and LCD display outputs at the same time.
+
+Make sure your Raspberry Pi OS is up-to-date before you begin working with the Touch Display to reduce the risk of errors or driver issues.
 
 === Board Support
 
-The DSI display is designed to work with all models of Raspberry Pi, however early models that do not have mounting holes (the Raspberry Pi 1 Model A and B) will require additional mounting hardware to fit the HAT-dimensioned stand-offs on the display PCB.
+The DSI display is designed to work with all models of Raspberry Pi, however early models without mounting holes (the Raspberry Pi 1 Model A and B) will require additional mounting hardware to fit the HAT-sized stand-offs on the display PCB.
+
+=== Connecting a Raspberry Pi
+
+The following image shows how to attach a Raspberry Pi directly to the back of the Touch Display using its standoffs. It also shows the connections you need to make between Raspberry Pi and the display's PCB. You can also mount the touch display in a separate chassis if you have one available. The connections will remain the same, though you may need longer cables depending on the chassis you use.
+
+image::images/GPIO_power-500x333.jpg[DSI Display pinout]
+
+==== Connecting the display cable
+
+Connect one end of the ribbon cable to the `RPI-DISPLAY` port on the Touch Display PCB so the side with the blue stripe faces the display.
+
+// Close-up image here?
```

Comment:
I'd like to get this merged. So lets not leave placeholders in PRs that aren't just drafts. Tighten this section up and describe how to do the connection without the images. Then we can open an issue to add images to the section after it is merged. "Series of images needed to illustrate display connection needed" or something like that.

Reply:
One of the big things about moving from a corporate-only repo to a public one is breaking old habits. This is one of those habits.

Consider this noted and I'll do better.